Tools/Pipeline/MainWindow.cs

```diff
@@ -62,13 +62,11 @@ public MainWindow()
         protected override void OnClosing(System.ComponentModel.CancelEventArgs e)
         {
             e.Cancel = !PipelineController.Instance.Exit();
-            base.OnClosing(e);
-        }
 
-        public override void Close()
-        {
-            Application.Instance.Quit();
-            base.Close();
+            if (!e.Cancel)
+                Xwt.Application.Exit();
```

Comment:
Xwt starts a Task with DragAndDrop operation so I needed to Exit the app from it, otherwise it wouldn't exit.